Makefile

```diff
@@ -219,7 +211,7 @@ PSLITE:
 $(DMLC_CORE)/libdmlc.a: DMLCCORE
 
 DMLCCORE:
-	+ cd $(DMLC_CORE); make libdmlc.a USE_SSE=$(USE_SSE) config=$(ROOTDIR)/$(config); cd $(ROOTDIR)
```

Comment:
Yes we will need to pass the flag to the DMLC_CORE make file. I did not want to do architecture resolution in multiple places